### ISSUE TITLE
KL Loss.

### ISSUE THREAD

**Issue by Prasanna1991** [wiseodd/generative-models#40]
Hi, I just noticed that the KL Loss in the VAE paper would look like this: 
0.5 * torch.sum(torch.exp(logVar) + mean ** 2 - 1. - logVar)

And here, the KL Loss is:
torch.mean(0.5 * torch.sum(torch.exp(logVar) + mean ** 2 - 1. - logVar, 1))

What's your thought in this?

**Comment by wiseodd** [comment]
It's because the `recon_loss` is averaged, therefore it also has to be averaged to be on the same scale.
You can use sum, but you have to change your learning rate.

**Comment by Prasanna1991** [comment]
Thanks. I think this explains it well.
